### PR TITLE
Fix Edge-LB Secret Creation Line and Add Known v0.1.9 CLI Issue

### PR DESCRIPTION
**Pull request by drewkerrigan**

### CODE REVIEW

File: pages/services/edge-lb/0.1.9/release-notes/index.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Shortlist:
 
 ## Known Limitations
 
+* The Edge-LB v0.1.9 CLI requires that all pool definitions are contained within a single YAML or JSON file. Pools cannot be configured independently because of this limitation.
```
Comment (asridharan) [inline pages/services/edge-lb/0.1.9/release-notes/index.md]: Do we need to highlight that we have removed this limitation from 1.0?

File: pages/services/edge-lb/1.0.0/release-notes/index.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ Shortlist:
 
 ## Known Limitations
 
+* The Edge-LB v0.1.9 CLI requires that all pool definitions are contained within a single YAML or JSON file. Pools cannot be configured independently because of this limitation.
```
Comment (asridharan) [inline pages/services/edge-lb/1.0.0/release-notes/index.md]: Why are we referring to `v0.1.9` here? A bit odd.

File: pages/services/edge-lb/0.1.9/release-notes/index.md (hunk 1 above)
Comment (drewkerrigan) [reply pages/services/edge-lb/0.1.9/release-notes/index.md]: In the 1.0.0 noteworthy changes, we have this line: `- New CLI command layout, ability to create, modify, and delete individual pools`, is that sufficient?

File: pages/services/edge-lb/1.0.0/pool-configuration/v2-examples/index.md [hunk 3]
```diff
@@ -325,6 +325,50 @@ To direct traffic based on the hostname to multiple backends for a single port (
...
+    "backends": [{
+      "name": "default",
+      "protocol": "HTTP",
+      "services": [{
+        "marathon": {
+          "serviceID": "/app-v1"
+        },
+        "endpoint": {
+          "portName": "web",
+          "miscStr": "weight 20"
```
Comment (asridharan) [inline pages/services/edge-lb/1.0.0/pool-configuration/v2-examples/index.md]: nice !! 👏

**Pull request merged**